src/validations/tasks.js

```diff
@@ -0,0 +1,25 @@
+import Joi from 'joi';
+
+const validateCreation = (req, res, next) => {
+  const tasksValidation = Joi.object({
+    name: Joi.string().min(3).max(50).required(),
+    description: Joi.string().min(3).max(50).required(),
+    project_id: Joi.required(),
+    hours: Joi.number().required(),
+  });
+
+  const validation = tasksValidation.validate(req.body);
+
+  if (validation.error) {
+    return res.status(400).json({
+      message: `There was an error: ${validation.error.details[0].message}`,
+      data: undefined,
+      error: true,
+    });
+  }
+  return next();
+};
+
+export default {
+  validateCreation,
+};
```

Comment:
if we are exporting this with default, there is no need to export it as an object. Export the plain method instead
```suggestion
export default validateCreation;
```